### ISSUE TITLE
Blocks displaying on the screen

### ISSUE THREAD

**Issue by Derrreks** [Derrreks/SideScroller#20]
Something we need to fix is that right now the block don't correctly display onto the screen. It starts putting the blocks on the screen from the top left corner which means that if the block size is too big, the bottom left corner with the player may not appear. We need to make it so that it begins painting it from the bottom left or some approach like that so that the game always starts in the bottom left corner of the level.

**Comment by pierfier** [comment]
Yeah, you can do that by starting the for loop from the end of the array
and go towards the beginning.

**Comment by Derrreks** [comment]
If you want to give it a go and try it fix it you can

**Comment by Derrreks** [comment]
So pierre did you get the camera to work? How is it coming?

**Comment by pierfier** [comment]
Yeah, im trying.

**Comment by Derrreks** [comment]
don't worry about it. I finished it it's all done

**Comment by Derrreks** [comment]
i updated the camera and everything about an hour ago it all works

**Comment by pierfier** [comment]
Excellent!

**Comment by pierfier** [comment]
I will try it out

**Comment by Derrreks** [comment]
It works perfectly on mine so it should be good

**Comment by pierfier** [comment]
For some reason, the menu stays and the blocks are displayed underneath it.

**Comment by Derrreks** [comment]
the menu stays? the background blocks should stay but that's it

**Comment by Derrreks** [comment]
try again i think i found the problem. I didn't update all the files right

**Comment by Derrreks** [comment]
okay i'm getting the error too, i'll work on it

**Comment by Derrreks** [comment]
okay all should be well now

**Comment by Derrreks** [comment]
I also just formatted most of the classes and restructed the whole program
